src/dev-spaces-preview/azext_dev_spaces_preview/custom.py

```diff
@@ -50,15 +49,17 @@ def ads_use_dev_spaces(cluster_name, resource_group_name, space_name='default',
         if retCode != 0:
             return
 
+    if space_name is None:
+        space_name = 'default'
+
     retCode = subprocess.call(
         [azds_cli, 'space', 'select', '--name', space_name], stderr=PIPE)
     if retCode == 0:
         return
 
     create_space_arguments = [azds_cli, 'space', 'create', '--name', space_name]
-    if parent_space_name is not None:
-        create_space_arguments.append('--parent')
-        create_space_arguments.append(parent_space_name)
+    if prompt:
+        pass
```

Comment:
This doesn't do anything......

Reply:
Yes. We want to have it passed from azure-cli, so added it in the signature. Will add the functionality when the underlying cli (azds) will support it, in the next release.